nibabel/deprecated.py

```diff
@@ -30,14 +33,14 @@ class ModuleProxy:
     module.
     """
 
-    def __init__(self, module_name):
+    def __init__(self, module_name: str):
```

Comment:
```suggestion
    def __init__(self, module_name: str) -> None:
```
I realize having the type annotation for *module_name* makes this addition functionally redundant, but if you don't disagree, I think it's better to be explicit.